extensions/trace-propagators/README.md

```diff
@@ -5,3 +5,18 @@ OpenTelemetry Contrib Trace Propagators
 
 [javadoc-image]: https://www.javadoc.io/badge/io.opentelemetry/opentelemetry-contrib-trace-propagators.svg
 [javadoc-url]: https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-contrib-trace-propagators
+
+This project contains [trace propagators](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context/api-propagators.md) that support
+3rd-party propagation formats.  These propagators are here for convenience and are not officially
+supported by the OpenTelemetry Java maintainers.
+
+Issues/support for these propagators is only provided as a minimal "best effort", and critical
+bugs should be filed with the respective vendors themselves.
+
+* AWS X-Ray (file an issue here and mention @anuraaga)
+* OpenTracing (file an issue here and mention @carlosalberto)
+* B3 (file an issue on [the OpenZipkin repo](https://github.com/openzipkin/b3-propagation) that points here)
+* Jaeger (file an issue on [the Jaeger repo](https://github.com/jaegertracing/jaeger) that points here)
```

Comment:
I thought we were on the hook to support these 2. Has that changed?

Reply:
I think that's up to us to decide...but the discussion in #2067 led me to believe that this entire subproject was external/contrib/3rd party.  If these are first-class supported exporters, should we move them into the main codebase?

Reply:
I would very much prefer if we weren't having to support these propagators, but it's unclear to me whether these are considered 1st-class or 3rd-party or somewhere in between.

Reply:
I put a note in the SIG agenda for Wed. about the `TraceMultiPropagator`, so maybe we can figure this out then too?

Reply:
It's pretty explicit that they are official

https://github.com/open-telemetry/opentelemetry-specification/blob/84d2b533eb7f1120b476b2ba7f301e467c890304/specification/context/api-propagators.md#propagators-distribution

Natural too since without them, no one can migrate to OpenTelemetry :P I think B3 and Jaeger were here in this package without much consideration to how supports them, more just something like "why not keep all the propagators together". But we're definitely on the hook for them.

I'd be ok with moving those into `opentelemetry-api-trace` if the wording here would make the OTel support aspect convoluted.

Reply:
I'll propose changing the wording here and will file a separate issue for moving B3 and Jaeger to `opentelemetry-api-trace` to be peers of `W3CTraceContextPropagator`.

Reply:
FYI, still not sure where this is going to land:  https://github.com/open-telemetry/oteps/issues/142 and whether it applies to propagators or not, and what set of propagators.